util/readpass.c

```diff
@@ -82,11 +82,14 @@ readpass(char ** passwd, const char * prompt,
 		/* Read directly from stdin. */
 		readfrom = stdin;
 		break;
-	default:
+	case 1:
 		/* Try to open /dev/tty; if that fails, read from stdin. */
 		if ((readfrom = fopen("/dev/tty", "r")) == NULL)
 			readfrom = stdin;
 		break;
+	default:
+		warn0("readpass does not support devtty=%i", devtty);
```

Comment:
It means the same thing, but normally we use `%d` rather than `%i` in printf strings.

Reply:
oh right, that's a hold-over from when I was teaching C to first-year students... I recommended `%i` for ints and `%f` for floats, because when you say "decimal" to a first-year engineer, they think you mean a number which contains a decimal point (e.g., 2.13).